default_app/default_app.js

```diff
@@ -1,16 +1,15 @@
 const electron = require('electron')
-const app = electron.app
-const BrowserWindow = electron.BrowserWindow
+const {app, BrowserWindow} = electron
```

Comment:
Similar comment here, perhaps just make this `const {app, BrowserWindow} = require('electron')` and remove line 1?